### PR TITLE
Improving the look of the Style Guide.

### LINKED ISSUE #490: Improve the look of the Style Guide

### PR DESCRIPTION
**Pull request by vazquez**
Fixes #490

### CODE REVIEW

File: src/pages/style-guide/style-guide.less [hunk 1]
```diff
@@ -1,5 +1,17 @@
 #style-guide {
-  padding: 10px 20px;
+  padding: 30px 0;
+  background: @shadowPlum;
+  background: -webkit-linear-gradient(top, #141C29, #31325F);
```
Comment (gvn) [inline src/pages/style-guide/style-guide.less]: I think autoprefixer should handle the browser prefixes for you.

File: src/pages/style-guide/style-guide.jsx [hunk 2]
```diff
@@ -64,85 +64,128 @@ var StyleGuide = React.createClass({
...
-          <div className="color sapphire"><span>@sapphire</span></div>
-          <div className="color teal"><span>@teal</span></div>
-          <div className="color slate"><span>@slate</span></div>
-          <div className="color heatherGrey"><span>@heatherGrey</span></div>
-          <div className="color lightGrey"><span>@lightGrey</span></div>
-          <div className="color softGrey"><span>@softGrey</span></div>
-          <div className="color plum"><span>@plum</span></div>
-          <div className="color shadowPlum"><span>@shadowPlum</span></div>
-          <div className="color brick"><span>@brick</span></div>
+        <div id="style-guide-container">
```
Comment (gvn) [inline src/pages/style-guide/style-guide.jsx]: Usually I avoid nesting IDs (really I generally try to use them as little as possible). 

In this case I'd use `class="wrapper"` or `class="container"` with a `> .wrapper` selector under `#style-guide`. Multiple ID selectors are usually an indicator that too many IDs are being used.

**Comment by gvn** [comment]
A couple tweaks needed, but looking really nice!

**Comment by vazquez** [comment]
All ready for you @gvn! :)

**Comment by gvn** [comment]
Looks good, but hold off on merging until after snippet launches. :+1:

**Comment by vazquez** [comment]
Perfect, will do!

**Comment by vazquez** [comment]
@gvn Is this good to merge now?

**Comment by xmatthewx** [comment]
Should we keep our code freeze for today? In case we need to land a hotfix?

**Comment by vazquez** [comment]
@xmatthewx Okay cool :)

**Comment by gvn** [comment]
I've already merged in dark UI...I think we can cherry pick hotfixes.

**Comment by xmatthewx** [comment]
Great. Fine by me.

**Comment by vazquez** [comment]
I'm merging then :boom:

**Comment by gvn** [comment]
:metal:

**Pull request merged**